.vscode/launch.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
then people have to use vscode to do the development?

Reply:
They're not required, but it's a handy thing, so they don't have to create one all the time.